proposals/0075-precompile-for-secp256r1-sigverify.md

```diff
@@ -0,0 +1,381 @@
+---
+simd: "0075"
+title: Precompile for verifying secp256r1 sig.
+authors:
+  - Orion (Bunkr)
+  - Jstnw (Bunkr)
+  - Dean (Web3 Builders Alliance)
+category: Standard
+type: Core
+status: Draft
+created: 2024-02-27
+feature: (fill in with feature tracking issues once accepted)
+supersedes: "0048"
+---
+
+## Summary
+
+Adding a precompile to support the verification of signatures generated on
+the secp256r1 curve. Analogous to the support for secp256k1 and ed25519
+signatures that already exists in form of the
+`KeccakSecp256k11111111111111111111111111111` and
+`Ed25519SigVerify111111111111111111111111111` precompiles.
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL
```

Comment:
Why is this a part of the SIMD? This is from the template as guidance on how to write the SIMD.

Reply:
Trent wanted me to reference RFC 2119 and it mandates the following: 
`Authors who follow these guidelines
 should incorporate this phrase near the beginning of their document:`

Reply:
Nudging this, would love to sort this out @jacobcreech 🤝